webapp/app.py

```diff
@@ -100,9 +110,49 @@ def api():
     Show the static api page
     """
 
+    base_url = "https://raw.githubusercontent.com/maas/maas-openapi-yaml/main"
+    yaml_file = "openapi2.yaml"
+
+    try:
+        definition = request.urlopen(f"{base_url}/{yaml_file}")
+    except error.HTTPError or error.URLError:
+        # Fall back to local yaml file if hosted yaml is unreachable
+        definition = open(yaml_file)
```

Comment:
The other option is to download the file during e.g. the Docker build when the site is released and then always use the download file. This would always only be as up-to-date as the latest release, if that would be acceptable.

Reply:
I think that should be fine really, since this spec is just meant to document the 3.3 API (3.3 RC1 released today)